Index: src/test/java/pages/PageTest.java
===================================================================
diff --git a/src/test/java/pages/PageTest.java b/src/test/java/pages/PageTest.java
deleted file mode 100644
--- a/src/test/java/pages/PageTest.java	(revision bb803214c4293b495ee03d74fff2db7ad43084b7)
+++ /dev/null	(revision bb803214c4293b495ee03d74fff2db7ad43084b7)
@@ -1,4 +0,0 @@
-package pages;
-
-public class PageTest {
-}
