Index: src/test/java/pages/PageTest.java
===================================================================
diff --git a/src/test/java/pages/PageTest.java b/src/test/java/pages/PageTest.java
deleted file mode 100644
--- a/src/test/java/pages/PageTest.java	(revision 19f3c9ca9d48dd2e796c29ba7d51c8f86c3dbb7c)
+++ /dev/null	(revision 19f3c9ca9d48dd2e796c29ba7d51c8f86c3dbb7c)
@@ -1,4 +0,0 @@
-package pages;
-
-public class PageTest {
-}
